### ISSUE TITLE
accessibility with font awesome

### ISSUE THREAD

**Issue by mpdehaan** [sourceoptics/source_optics#120]
The font awesome icons we use should have some kind of alt text

**Comment by mpdehaan** [comment]
reference: https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility
